docs/developer-guide/partition-data.rst

```diff
@@ -86,33 +137,3 @@ For example implementations of a custom partitioner, see
 `Built for realtime: Big data messaging with Apache Kafka, Part 2 <https://www.javaworld.com/article/3066873/big-data/big-data-messaging-with-kafka-part-2.html>`__
 and `Apache Kafka Foundation Course - Custom Partitioner <https://www.learningjournal.guru/courses/kafka/kafka-foundation-training/custom-partitioner/>`__.
 
-.. _ensure-co-partitioning:
-
-Ensure Data Co-partitioning
-***************************
-
-If your join inputs aren't co-partitioned, you must ensure it manually
-by re-keying the data on one side of the join.
-
-For example, in a stream-table join, if a stream of user clicks is keyed by
-``pageId``, but a table of user profiles is keyed by ``userId``, one of the
-two inputs must be re-keyed (re-partitioned). Which of the two should be re-keyed
-depends on the situation.
-
-If the stream has very high volume, you may not want to re-key it,

```

Comment:
I've removed this bit about rekeying the table, as we don't support rekeying a table :)